third-party/imgui/imgui.cpp

```diff
@@ -6455,10 +6455,12 @@ bool ImGui::SliderBehavior(const ImRect& frame_bb, ImGuiID id, float* v, float v
                 new_value = ImLerp(v_min, v_max, normalized_pos);
             }
 
-            // Round past decimal precision
+            // Round past decimal precision, verify that it remains within min/max range
             new_value = RoundScalar(new_value, decimal_precision);
             if (*v != new_value)
             {
+                if (new_value >v_max )  new_value = v_max;
```

Comment:
Shouldn't these boundaries be set outside the if?

Reply:
It will work both ways. Imo this is micro-optimization but the `if (*v != new_value)` branch will be invoked randomly, so those rectifications will be less frequent

Reply:
Done